ui/components/schema/structureView.css

```diff
@@ -50,3 +50,24 @@
 .structure-item-row {
     margin-top: 5px;
 }
+.flexWrapper {
```

Comment:
Why is this css added to the structure view? This looks duplicate to https://github.com/aerogear/data-sync-ui/blob/master/ui/components/schema/schemaContainer.css

Reply:
These styles are not used anywhere in `StructureView.js`.